src/dotnet-scaffolding/dotnet-scaffold-aspnet/Models/ViewModel.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+namespace Microsoft.DotNet.Tools.Scaffold.AspNet.Models;
+
+internal class ViewModel : CrudModel
+{
+    public string GetInputClassType(string inputType)
+    {
+        if (string.Equals(inputType, "bool", StringComparison.OrdinalIgnoreCase))
```

Comment:
You could do:

`return string.Equals(inputType, "bool", StringComparison.OrdinalIgnoreCase) ? "form-check-input" : "form-control"`